src/ontology/edits/oeo-physical.omn

```diff
@@ -1132,21 +1145,24 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/385",
 Class: OEO_00000054
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Air is a portion of matter that consists of a composition of gases that forms the Earth's atmosphere."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Air is a gas mixture that consists of a composition of gases that forms the Earth's atmosphere."@en,
```

Comment:
_Air is a gas mixture ~~that consists of a composition of gases~~ that forms the Earth's atmosphere._
I think that this duplication in def is useless so we should delete it.